scrollingelement.js

```diff
@@ -28,9 +28,27 @@ if (!('scrollingElement' in document)) {
 			return isFrameset ? null : body;
 		};
 
-		Object.defineProperty(document, 'scrollingElement', {
-			'get': scrollingElement
-		});
+		if ('defineProperty' in Object) {
+			Object.defineProperty(document, 'scrollingElement', {
+				'get': scrollingElement
+			});
+		} else if ('__defineGetter__' in document) {
+			// Firefox < 4, Safari < 5, Opera < 12
+			document.__defineGetter__('scrollingElement', scrollingElement);
+		} else {
+			// fallback
+			document.scrollingElement = scrollingElement();
+			if ('attachEvent' in document) {
```

Comment:
I wanted the `document.scrollingElement = scrollingElement();` to run even if attachEvent isn't supported. I suppose that's just for really old non-IE browsers. I don't care strongly about it, but this would be equivalent and save indentation:

```js
} else {
    document.scrollingElement = scrollingElement();
    document.attachEvent && document.attachEvent(...);
```